app/controllers/judgement_controller.rb

```diff
@@ -0,0 +1,33 @@
+class JudgementController < ApplicationController
```

Comment:
If a controller name refers to a resource (even if it's not a model by itself) we should use plural forms.